_data/other.yml

```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://www.ulule.com/
+      twitter: ulule
+      facebook: ulule
```

Comment:
i believe that `ulule` is the company's [Facebook root or default page](https://www.facebook.com/business/help/331800410323820)